stories/qa/icon/icon.stories.html

```diff
@@ -113,8 +113,8 @@ <h2>Palette</h2>
 <section class="contentSection">
 	<h2>List</h2>
 	<div *ngFor="let icon of icons">
-		<span aria-hidden="true" class="lucca-icon icon-{{icon}}"></span>
-		<code class="code">icon-{{icon}}</code>
+		<span aria-hidden="true" class="lucca-icon icon-{{icon.icon}}"></span>
```

Comment:
Shame that we can't destruct a *ngFor

icon icon icon icon 👯 
<img src="https://media0.giphy.com/media/zXHZWGLWNQkrS/giphy.gif"/>